test/webgl/context-features.spec.js

```diff
@@ -76,3 +78,52 @@ test('webgl#caps#hasFeatures(WebGL2)', t => {
 
   t.end();
 });
+
+test('webgl#caps#canCompileGLGSExtension', t => {
+  const {gl} = fixture;
+  const oldNavigator = window.oldNavigator;
+
+  t.ok(typeof canCompileGLGSExtension === 'function', 'canCompileGLGSExtension defined');
+
+  // Non-IE version.
+  const getShaderParameterStub = sinon.stub(gl, 'getShaderParameter');
```

Comment:
I didn't realize we still have `sinon` as a dependency. For consistency with other vis.gl frameworks I would prefer that we use `makeSpy` from [probe.gl/test](https://uber-web.github.io/probe.gl/#/documentation/api-reference-testing/makespy).

```
import {makeSpy} from 'probe.gl/test';
... makeSpy(gl, '...');
```
should work similarly.

Reply:
Does probe.gl offer a stubbing mechanism? I was using sinon to override the getShaderParameter function so I can have it return true/false based on the test. makeSpy only allows me to observe call counts.

I could remove the sinon dependency and directly override the function if you'd prefer.